### PR TITLE
ASC-257 Add write to attached volume test

### PR DESCRIPTION
**Pull request by johnduarte**
This commit adds a test to attach a volume to an instance, create a
partition and filesystem on that volume, and verify you can write to it.

### CODE REVIEW

File: molecule/default/tests/test_write_to_attached_storage.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+            except Exception:
+                print ("\n Trying again in 5 seconds.....\n")
+                time.sleep(5)
+                continue
+            break
+# End Helpers copied from Cinder utils.py
+
+
+# fresh helpers
+def create_floating_ip(server, run_on_host):
```
Comment (phongdly) [inline molecule/default/tests/test_write_to_attached_storage.py]: Should this function be spit into two: creating floating IP and assigning the IP to a server? Or changing the method name to `create_and_assign_floating_ip`?

Comment (johnduarte) [reply molecule/default/tests/test_write_to_attached_storage.py]: It should definitely be split up. Thanks @phongdly

**Comment by johnduarte** [comment]
@phongdly I have updated this PR to remove the helpers that have already been included in pytest-rpc. The new helpers have been added as a discrete commit that can be reverted when the helpers are available in pytest-rpc. The floating-ip helper has been split into create and attach helpers.
Please let me know if you see anything I missed.

**Comment by johnduarte** [comment]
@phongdly the flake8 issues have been addressed and the test is passing on my MNAIO deployment:
```
    ============================= test session starts ==============================
    platform linux2 -- Python 2.7.12, pytest-3.6.3, py-1.5.4, pluggy-0.6.0 -- /home/ubuntu/rpc-openstack-system-tests/venv-molecule/bin/python
    using: pytest-3.6.3 pylib-1.5.4
    setuptools registered plugins:
      testinfra-1.12.0 at /home/ubuntu/rpc-openstack-system-tests/venv-molecule/local/lib/python2.7/site-packages/testinfra/plugin.py
      pytest-rpc-0.8.0 at /home/ubuntu/rpc-openstack-system-tests/venv-molecule/local/lib/python2.7/site-packages/pytest_rpc/__init__.pyc
    rootdir: /home/ubuntu/rpc-openstack-system-tests/molecules/molecule-rpc-openstack-post-deploy/molecule/default, inifile:
    plugins: testinfra-1.12.0, rpc-0.8.0
collected 39 items

    tests/test_write_to_attached_storage.py::test_volume_attached[ansible://infra1] PASSED [  2%]
    ...
    tests/test_for_acs-150.py::TestForRPC10PlusPostDeploymentQCProcess::test_verify_kibana_horizon_access_with_no_ssh[_testinfra_host0] SKIPPED [100%]

     generated xml file: /home/ubuntu/rpc-openstack-system-tests/molecules/molecule-rpc-openstack-post-deploy/molecule/default/molecule_test_results.xml
    ==================== 9 passed, 30 skipped in 130.40 seconds ====================
```

**Pull request merged**